### DISCUSSION TITLE
Store fonts locally

### DISCUSSION THREAD

**Discussion by mdbdb** [CaiJimmy/hugo-theme-stack#457] [Q&A] [answered]
Hello,

I love the theme and am currently customizing it for my blog. I am not a fan of embedding fonts externally from Google.
Is there any way to save fonts locally and embed them in the theme?

What changes do I need to make and where do the fonts need to be saved?

Thanks for the help,
mdbdb

**Comment by CaiJimmy** [comment, marked as answer]
> Is there any way to save fonts locally and embed them in the theme?

Google fonts are loaded through this file:
https://github.com/CaiJimmy/hugo-theme-stack/blob/7dab67b358b6419e1c182876d89cdfb49b9f5cc2/layouts/partials/footer/components/custom-font.html

Theoretically, you can download that CSS along with all woff2 files, and replace `src` with local links, and that should do the job.

**Reply by mdbdb** [reply]
Thanks a lot, that worked!
